src/managed/CommonManaged.props

```diff
@@ -24,7 +24,7 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' != 'netstandard2.0' ">
+  <PropertyGroup Condition=" '$(TargetFramework)' != 'netstandard2.0' And '$(TargetFramework)' != 'netcoreapp2.0' ">
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
```

Comment:
Out of curiosity, do you know why this project or the other managed projects need to disable implicit framework references?

Reply:
This project only targets netcoreapp2.0, so it doesn't disable implicit framework references.
The  `PlatformAbstraction` and `DependencyModel` projects disable it for netstandard1.3 and [include specific versions of packages](https://github.com/dotnet/core-setup/blob/master/src/managed/Microsoft.DotNet.PlatformAbstractions/Microsoft.DotNet.PlatformAbstractions.csproj), but I don't know why.